packages/pwa-kit-react-sdk/src/ssr/universal/components/with-correlation-id/index.js

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2022, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import React from 'react'
+import {useCorrelationId} from '../../hooks'
+
+const withCorrelationId = (Component) => {
```

Comment:
Please add a jsdoc style comment block for this component. 

We should probably used the `@private` indicator since we are only using it internally in react-rendering and main. This will allow use to make changes without having to worry about notifying others. On the other hand the hook is not private.